ax/models/random/base.py

```diff
@@ -135,7 +135,7 @@ def gen(
             if self.fallback_to_sample_polytope:
                 logger.info(
                     "Rejection sampling exceeded specified maximum draws."
-                    "Falling back on polytope sampler"
+                    "Falling back on polytope sampler instead of Sobol sampler."
```

Comment:
Looks like this is on the base class, which is shared between uniform & sobol samplers. 
```suggestion
                    "Falling back on HitAndRunPolytopeSampler instead of "
                    f"{self.__class__.__name__}."
```